### PR TITLE
{data,lib,vis}[GCCcore/11.2.0] OpenSceneGraph v3.6.5, librsvg v2.52.8, DCMTK v3.6.6

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.6-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ('CMake', '3.21.1')
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('LibTIFF', '4.3.0'),
+    ('libpng', '1.6.37'),
+    ('libxml2', '2.9.10'),
+    ('libiconv', '1.16'),
```
Comment (verdurin) [inline easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.6-GCCcore-11.2.0.eb:29]: Seems that it may need `OpenSSL`, based on the build failures I've seen.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 4 out of 6 (3 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/03bb40f44e4462099f34a5d6f72ab0f1 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 10 out of 12 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/1fd4bcf5808fe19fccb4c4675acad205 for a full test report.

File: easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.6-GCCcore-11.2.0.eb (hunk 1 above)
Comment (furstj) [reply easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.6-GCCcore-11.2.0.eb:30]: OpenSSL has been added to DCMTK package

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f72be27818b37f991be0270f6323ee87 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a1a07c38962a3d4f36c97e4b4dd98ba3 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @furstj!

**Pull request merged**